### DISCUSSION TITLE
Use x-nullable extensions when reading/writing OASv2

### DISCUSSION THREAD

**Discussion by ouvreboite** [microsoft/OpenAPI.NET#1250] [Ideas]
While OASv2 does not have built-in support for nullable, the `x-nullable` extension is often used.
Could it be supported for OASv2 serialization or deserialization? Maybe as an optional setting in the writer and reader?

**Comment by baywet** [comment]
hi @ouvreboite

Thank you for using the SDK and for reaching out.

This is something that was added to versions 2.X and 3.X of the library.

Let us know if you have any additional comments or questions.
